build/Wikibase/jobrunner-entrypoint.sh

```diff
@@ -3,25 +3,13 @@
 # Originally inspired by Brennen Bearnes jobrunner entrypoint
 # https://gerrit.wikimedia.org/r/plugins/gitiles/releng/dev-images/+/refs/heads/master/common/jobrunner/entrypoint.sh
 
-# Wait for the db to come up
-/wait-for-it.sh "$DB_SERVER" -t 300
-# Sometimes it appears to come up and then go back down meaning MW install fails
-# So wait for a second and double check!
-sleep 1
-/wait-for-it.sh "$DB_SERVER" -t 300
-
 kill_runner() {
 	kill "$PID" 2> /dev/null
 }
 trap kill_runner SIGTERM
 
 while true; do
-	if [ -e /shared/LocalSettings.php ]; then
```

Comment:
`/config/LocalSettings.php` is now required in the entry point for the container to run, so this check is no longer needed.